src/resources/postcss/components/_view-selector.pcss

```diff
@@ -0,0 +1,273 @@
+.tribe-events {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * View Selector
+	 *
+	 * Example (Select):
+	 * <div class="tribe-events-c-view-selector">
+	 *   <button
+	 *     class="tribe-events-c-view-selector__button"
+	 *     aria-controls="container-id"
+	 *     aria-expanded="false"
+	 *     aria-selected="false"
+	 *   >
+	 *     <span class="tribe-events-c-view-selector__button-icon tribe-common-svgicon tribe-common-svgicon--month"></span>
+	 *     <span class="tribe-events-c-view-selector__button-text">
+	 *       Month
+	 *     </span>
+	 *   </button>
+	 *   <div
+	 *     class="tribe-events-c-view-selector__content"
+	 *     id="container-id"
+	 *     aria-hidden="true"
+	 *   >
+	 *     <ul class="tribe-events-c-view-selector__list">
+	 *       <li class="tribe-events-c-view-selector__list-item">
+	 *         <a href="#" class="tribe-events-c-view-selector__list-item-link" data-js="tribe-events-view-link">
+	 *           <span class="tribe-events-c-view-selector__list-item-icon tribe-common-svgicon tribe-common-svgicon--month"></span>
+	 *           <span class="tribe-events-c-view-selector__list-item-text">
+	 *             Month
+	 *           </span>
+	 *         </a>
+	 *       </li>
+	 *       <li class="tribe-events-c-view-selector__list-item">
+	 *         <a href="#" class="tribe-events-c-view-selector__list-item-link" data-js="tribe-events-view-link">
+	 *           <span class="tribe-events-c-view-selector__list-item-icon tribe-common-svgicon tribe-common-svgicon--list"></span>
+	 *           <span class="tribe-events-c-view-selector__list-item-text">
+	 *             List
+	 *           </span>
+	 *         </a>
+	 *       </li>
+	 *     </ul>
+	 *   </div>
+	 * </div>
+	 *
+	 * Example (Tabs):
+	 * <div class="tribe-events-c-view-selector tribe-events-c-view-selector--tabs">
+	 *   ... (same as above)
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-c-view-selector {
+		position: relative;
+		width: 29px;
+
+		@media (--viewport-medium) {
+			width: 114px;
```

Comment:
Are we setting the selector to a specific view? Have we tried with texts that are not only the ones included in the designs? Thinking about translations here and how this can end up looking bad if those words change.

Reply:
Good question. No, this wouldn't allow for longer words. The designs have some odd spacing that I'd like to clear up with Matt/Rachel.

Reply:
We can come back to this, would you mind noting this in a ticket @juanfra  we need to review some of those design points.

Reply:
I have a ticket for tabs/select work: https://central.tri.be/issues/129298 I can have a look later, this shouldn't hold up the merge for QA.